### ISSUE TITLE
Only variables should be passed by reference at .../lib/private/cache/fileglobalgc.php#50

### ISSUE THREAD

**Issue by Siedlerchr** [owncloud/core#15010]
Hi,

since the upgrade to 8.0.2 I sometimes get the following error in my ocl log:
PHP Error:
Only variables should be passed by reference at `/home/xxxx/public_html/lib/private/cache/fileglobalgc.php#50` 

I have no idea what this means, my system config;

```
Apache-Version	2.4.10
PHP-Version	5.4.33
MySQL-Version	5.6.23
Architecture: x86_64
OS:  linux
Kernel-Version	2.6.32-042stab102.9
```

**Comment by karlitschek** [comment]
@icewind1991 can you have a look please?

**Comment by RealRancor** [comment]
Looks to me as a duplicate to: https://github.com/owncloud/core/issues/14932 which gets fixed by: https://github.com/owncloud/core/pull/14980

@Siedlerchr Please have a look at the search in  this issue tracker first for keywords like the "fileglobalgc" before posting a new issue to find such duplicates.

**Comment by Siedlerchr** [comment]
@RealRancor  I searched for the error message and could not found any related issue.
Sorry for posting duplicate.

**Comment by Xenopathic** [comment]
@RealRancor This is a slightly different issue, but has been fixed in master: 5bb1a131e42a4db1b64aab563d36dc996635a9dd. The fix is also in stable8, and so must be coming out with 8.0.3.
